src/scrape_up/timeanddate/day_in_history.py

```diff
@@ -0,0 +1,206 @@
+from urllib.request import Request, urlopen
+from bs4 import BeautifulSoup as soup
+
+
+class DayInHistory:
+
+    """
+    Create an instance of `DayInHistory` class.
+    
+    ```python
+    dayinhistory = DayInHistory()
+
+    ```
+    | Methods              | Details                                          |
+    | ---------------------|--------------------------------------------------|
+    | `.important_events()`| Returns the timezones of cites around the world  |
+    | `.holidays()`        | Returns the holidays on the specific day         |
+    | `.births()`          | Returns the important birthdays on that day      |
+    | `.deaths()`          | Returns the important deaths on that day         |
+
+    """
+
+    def __init__(self):
+        self.__scrape_page()
+
+    def __scrape_page(self):
+    
+        try:
+            url = "https://www.timeanddate.com/on-this-day/"
+            req = Request(url, headers={"User-Agent": "Mozilla/5.0"})
+
+            webpage = urlopen(req).read()
+            self.page_soup = soup(webpage, "html.parser")
+
+        except:
+            return None
+
+    def important_events(self):
+
+        """
+        Create an instance of `DayInHistory` class.\n
+
+        ```python
+        dayinhistory = DayInHistory()
+        dayinhistory.important_events()
+        ```
+
+        Return\n
+        ```js
+        {
+            "2008 Coup in Mauritania": "In the 6th coup in the North African country since 1978,
```

Comment:
This is not an appropriate way to return data. Return an array of objects, where each object looks like
```python
{
    "year":"2008",
    "event":"In the 6th coup in the North African country since 1978..."
}
```